maintainers/maintainer-list.nix

```diff
@@ -6149,10 +6149,10 @@
     githubId = 6455574;
     name = "Matt Votava";
   };
-  mwilsoninsight = {
-    email = "max.wilson@insight.com";
-    github = "mwilsoninsight";
-    githubId = 47782621;
+  maxwilson = {
```

Comment:
I think the commit message for this should be:
```
maintainers: mwilsoninsight -> maxwilson
```
then have an explanation in the body.